docs/testing.md

```diff
@@ -61,6 +61,13 @@ We use a combination of [Lighthouse-ci](https://github.com/GoogleChrome/lighthou
 
 Accessibility tests are executed as part of our CI/CD pipeline on each PR to the `main` branch, commit to the `main` branch, and PR into the `prod` branch.
 
+To run Lighthouse or pa11y locally, just run:
```

Comment:
I don't think this runs in the docker container because the config file is above the /backend directory. For now, I think that shouldn't block the pr.  You can accept this if that's right, if could you help me trouble shoot locally?
```suggestion
To run Lighthouse or pa11y locally, install the dependencies on your machine and outside the docker container, run the following commands.
```